app/javascript/sidebar-widget/SidebarWidget.jsx

```diff
@@ -27,22 +27,24 @@ class SidebarWidget extends Component {
     });
   }
 
-  getSuggestedUsers() {
+  async getSuggestedUsers() {
```

Comment:
Unfortunately this doesn't work. I'm not sure how to enable it but right now `async/await` doesn't work, though oddly enough it's used in some of the tests we have. 

If I open the tag page which uses this widget I get: `regeneratorRuntime is not defined` as an error:

![Screenshot 2020-03-08 at 11 49 13 AM](https://user-images.githubusercontent.com/146201/76161347-f252e780-6132-11ea-9b28-f366ba0e2364.png)

I think @nickytonline might know how to deal with this and enable the new syntax

Reply:
I saw babel-polyfill somewhere so I thought i can use it. I can switch to then/catch?

Reply:
Thanks for the PR @frkozbk. For the time being we’re not using async/await since the code generated is a large chunk of code using generators and gets repeated anywhere async/await is used. We have upgrading to Babel 7 on our roadmap and plan to exclude using regenerator runtime as the browsers we support all support async/await as far as I know.

Reply:
Having said that I could disable it potentially in Babel 6. Not sure if it’s supported in that version.

Reply:
I changed with then/catch.